<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoInfoDialog</class>
 <widget class="QDialog" name="VideoInfoDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>554</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Video information</string>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Video title</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_author">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_keywords">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Keywords</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_length">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Length</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_publish_date">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Publish date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_thumbnail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <zorder>lbl_thumbnail</zorder>
  <zorder>lbl_title</zorder>
  <zorder>lbl_author</zorder>
  <zorder>textBrowser_description</zorder>
  <zorder>lbl_keywords</zorder>
  <zorder>lbl_length</zorder>
  <zorder>lbl_publish_date</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
